Source/Fuse.Controls.Primitives/Image.Visual.uno

```diff
@@ -5,16 +5,32 @@ using Fuse.Drawing;
 using Fuse.Internal;
 using Fuse.Elements;
 using Fuse.Nodes;
+using Fuse.Resources;
+using Fuse.Resources.Exif;
 
 namespace Fuse.Controls
 {
 	public partial class Image
 	{
+		public bool IgnoreExif { get; set; }
```

Comment:
What's the purpose of this?

Reply:
for testing. haven't done a proper cleanup yet.